web/src/components/nomination/NominateForm.vue

```diff
@@ -0,0 +1,88 @@
+<template>
+  <v-form class="my-3">
+    <name-input
+      v-model="form.speakerName"
+    />
+    <v-text-field
+      v-model="form.speakerEmail"
+      label="Email"
+      outlined
+    />
+    <job-input
+      v-model="form.firstJob"
+    />
+    <location-input
+      v-model="form.location"
+    />
+    <v-textarea
+      v-model="form.speakerBio"
+      label="Speaker Bio"
+      hint="A brief description of the nominee"
+      outlined
+    />
+    <topics-input v-model="form.topics" />
+
+    <submitter-input
+      v-model="form.submitterInput"
+    />
+    <v-checkbox
+      v-model="form.permission"
+      class="mt-0"
+      label="I have the speaker's permission to submit her information to the SpeakHer database."
+    />
+    <v-btn color="primary">
+      Submit
+    </v-btn>
+  </v-form>
+</template>
+
+<script>
+import NameInput from '@/components/nomination/NameInput.vue';
+import JobInput from '@/components/nomination/JobInput.vue';
+import LocationInput from '@/components/nomination/LocationInput.vue';
+import SubmitterInput from '@/components/nomination/SubmitterInput.vue';
+import TopicsInput from '@/components/nomination/TopicsInput.vue';
+
+export default {
+  components: {
+    NameInput,
+    JobInput,
+    LocationInput,
+    SubmitterInput,
+    TopicsInput,
```

Comment:
I'm not sure I understand the design being used here. Wouldn't components usually be reusable parts that we can plug and play in other parts of the code? Like `TextInput` or `SelectInput`.

These components feel more like "sections" of the form, is that right? What was the intention behind breaking these parts of the form in components?

Reply:
Yeah, just breaking it into sections to make the code more modular. We might be able to visit smaller granularity as a future refactor, I'd be curious to see how that might work. In the meantime v-text-input is the component we get from Vuetify.